stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -419,7 +420,12 @@ public class MessageListViewModel(
             messagesState = messagesState.copy(unreadCount = getUnreadMessageCount())
         }
 
-        chatDomain.markRead(channelId).enqueue()
+        if (ToggleService.isEnabled(ToggleService.TOGGLE_KEY_OFFLINE)) {
+            val (channelType, channelId) = channelId.cidToTypeAndId()
```

Comment:
@filbabic Please confirm if `channelId` here is cid or not.

Reply:
@amitkma 

`this.channelId` is cid. channelId here is just `id` (without the type).

Can you please make sure this works based on the logic that should be here? Also - if you're changing anything, can you update the value name here (in the destructured declaration) to `id` from `channelId`?

Reply:
Thanks for the confirmation.